doc/install.rst

```diff
@@ -14,34 +14,62 @@ Installing
     <https://forum.generic-mapping-tools.org/c/questions/pygmt-q-a>`__.
 
 
+Quickstart
+----------
+
+The fastest way to install PyGMT is with the
+`conda <https://docs.conda.io/projects/conda/en/latest/user-guide/index.html>`__
+package manager which takes care of setting up a virtual environment, as well
+as the installation of GMT and all the dependencies PyGMT depends on::
+
+    conda create --name pygmt --channel conda-forge pygmt
+
+To activate the virtual environment, you can do::
+
+    conda activate pygmt
+
+After this, check that everything works by running the following in a Python
+interpreter. If you are using Jupyter, make sure you have connected to the
+'pygmt' kernel before running the below::
```

Comment:
I saw you mentioned "pygmt" kernel in the forum, but I never see a "pygmt" kernel, and everything works well for me (on macOS).

![image](https://user-images.githubusercontent.com/3974108/107735257-4b28e980-6ccd-11eb-85a9-6188041b84d7.png)

> make sure you have connected to the 'pygmt' kernel before running.

This sentence may make users like me very confusing.

Reply:
Ah right, I might take that out then. It's only when people don't launch jupyter in the 'pygmt' conda environment (i.e. after `conda activate pygmt`) that they will need to connect specifically to the 'pygmt' kernel (and perhaps in some special cases).

Reply:
```suggestion
interpreter (e.g. in a Jupyter notebook)::
```

Reply:
I also don't the the pygmt kernel when I launch the jupyter in the base environment.